src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -114,6 +115,12 @@ public function getName()
 
     private static function escape(Environment $env, string $s): string
     {
+        // Twig 3.10 and above
+        if (class_exists(EscaperRuntime::class) && null !== $escaperRuntime = $env->getRuntime(EscaperRuntime::class)) {
```

Comment:
Do we actually need the `class_exists()` check?

Reply:
Without we get this error:

> Twig\Error\RuntimeError: Unable to load the "Twig\Runtime\EscaperRuntime" runtime.

Reply:
Right, so the `null` check is unnecessary then.

Reply:
Ah indeed, I was confused by the behaviour of the mock we used before.

Reply:
this is what allows detecting Twig 3.10+